pkg/authorization/cache/readonlyclusterpolicybindingcache.go

```diff
@@ -0,0 +1,108 @@
+package cache
```

Comment:
i think you can do this like clients, so names like:

pkg/authorization/cache/clusterpolicybinding.go is fine.

the cache suffix is redundant

the readonly is obvious when in there.